# Individual touch event data
int32 touch_id           # Unique touch identifier
float32 x                # X coordinate in screen space
float32 y                # Y coordinate in screen space
float32 depth            # Depth value in meters
float32 distance         # Distance from plane in meters
bool is_touch            # Whether this is a valid touch

